splunk_connect_for_snmp_poller/manager/realtime/interface_mib.py

```diff
@@ -37,12 +39,14 @@ class InterfaceMib:
     METRIC_VALUE_KEY = "_value"
     METRIC_TYPE_KEY = "metric_type"
     IF_MIB_METRIC_PREFIX = "sc4snmp.IF-MIB."
+    NON_METRIC_IDENTIFIER = "ifDescr"
+    MONGO_IDENTIFIER = "IF-MIB"
```

Comment:
Is it some string constant that will be placed to Mongo? The name should be self explanatory.